### PR TITLE
doc: update vault-helm for 0.2.0

### PR DESCRIPTION
**Pull request by jasonodonnell**
In anticipation for the upcoming Vault Helm 0.2.0 release, the documentation has been updated to include new features and expand on examples.

### CODE REVIEW

File: website/source/docs/platform/k8s/helm.html.md [hunk 1]
```diff
@@ -173,6 +221,14 @@ and consider if they're appropriate for your deployment.
           disktype: ssd
         ```
 
+  * `extraLabels` (`string`) - This value defines additional labels for server pods. This should be a formatted as a multi-line string.
```
Comment (kalafut) [inline website/source/docs/platform/k8s/helm.html.md]:
```suggestion
  * `extraLabels` (`string`) - This value defines additional labels for server pods. This should be formatted as a multi-line string.
```

File: website/source/docs/platform/k8s/helm.html.md [hunk 2]
```diff
@@ -186,6 +242,28 @@ and consider if they're appropriate for your deployment.
     - `enabled` (`boolean: true`) - When set to `true`, a Kubernetes service will be created for Vault.
 
     - `clusterIP` (`string`) - ClusterIP controls whether an IP address (cluster IP) is attached to the Vault service within Kubernetes.  By default the Vault service will be given a Cluster IP address, set to `None` to disable.  When disabled Kubernetes will create a "headless" service.  Headless services can be used to communicate with pods directly through DNS instead of a round robin load balancer.
+    
+    - `port` (`int: 8200`) - Port on which Vault server is listening inside the pod.
+
+    - `targetPort` (`int: 8200`) - Port on which the service is listening.
+
+    - `annotations` (`string`) - This value defines additional annotations for the service. This should be a formatted as a multi-line string.
```
Comment (kalafut) [inline website/source/docs/platform/k8s/helm.html.md]:
```suggestion
    - `annotations` (`string`) - This value defines additional annotations for the service. This should be formatted as a multi-line string.
```

File: website/source/docs/platform/k8s/helm.html.md [hunk 3]
```diff
@@ -186,6 +242,28 @@ and consider if they're appropriate for your deployment.
...
+
+        ```yaml
+        annotations: |
+          "sample/annotation1": "foo"
+          "sample/annotation2": "bar"
+        ```
+
+ * `serviceAccount` - Values that configure the Kubernetes service account created for Vault.
+
+    - `annotations` (`string`) - This value defines additional annotations for the service account. This should be a formatted as a multi-line string.
```
Comment (kalafut) [inline website/source/docs/platform/k8s/helm.html.md]:
```suggestion
    - `annotations` (`string`) - This value defines additional annotations for the service account. This should be formatted as a multi-line string.
```

File: website/source/docs/platform/k8s/helm.html.md [hunk 4]
```diff
@@ -71,11 +71,22 @@ and consider if they're appropriate for your deployment.
...
+  * `imagePullSecrets` (`string: ""`) - Defines secrets to be used when pulling images from private registries.
+
+      - `name`: (`string: required`) - 
+      Name of the secret containing files required for authentication to private image registries.
 
   * `tlsDisable` (`boolean: true`) - When set to `true`, changes URLs from `https` to `http` (such as the `VAULT_ADDR=http://127.0.0.1:8200` environment variable set on the Vault pods).
 
 * `server` - Values that configure running a Vault server within Kubernetes.
 
+  * `securityContext` - Values that configure Vault pods security context.
```
Comment (malnick) [inline website/source/docs/platform/k8s/helm.html.md]: Woot, happy to see this in here!

File: website/source/docs/platform/k8s/helm.html.md (hunk 1 above)
Comment (jasonodonnell) [reply website/source/docs/platform/k8s/helm.html.md]: I thought I fixed these, most have forgot.  Thanks!

**Pull request merged**